biojava-core/src/test/java/org/biojava/nbio/core/sequence/io/GenbankReaderTest.java

```diff
@@ -333,6 +330,36 @@ public void readSequenceWithZeroSpanFeature() throws IOException, CompoundNotFou
 		assertEquals(Strand.NEGATIVE, fLocation.getStrand());
 	}
 
+	/**
+	 * Biojava fails to parse anticodon and transl_except feature qualifiers when they line wrap.
+	 * https://github.com/biojava/biojava/issues/843
+	 * Except file NC_018080.gb from https://www.ncbi.nlm.nih.gov/nuccore/NC_018080
+	 */
+	@Test
+	public void testGithub843() throws Exception {
+		CheckableInputStream inStream = new CheckableInputStream(this.getClass().getResourceAsStream("/NC_018080.gb"));
+		assertNotNull(inStream);
+
+		GenbankReader<DNASequence, NucleotideCompound> genbankDNA
+				= new GenbankReader<>(
+				inStream,
+				new GenericGenbankHeaderParser<>(),
+				new DNASequenceCreator(DNACompoundSet.getDNACompoundSet())
+		);
+
+		LinkedHashMap<String, DNASequence> dnaSequences = genbankDNA.process();
+		assertNotNull(dnaSequences);
+
+		DNASequence dna = new ArrayList<>(dnaSequences.values()).get(0);
+		assertNotNull(dna);
+
+		for (Iterator<FeatureInterface<AbstractSequence<NucleotideCompound>, NucleotideCompound>> it = dna.getFeaturesByType("tRNA").iterator(); it.hasNext();) {
+			FeatureInterface<AbstractSequence<NucleotideCompound>, NucleotideCompound> tRNAFeature = it.next();
+			String anticodon = tRNAFeature.getQualifiers().get("anticodon").get(0).getValue();
+			assertFalse(anticodon.contains(" "));
```

Comment:
How about also asserting that the string is exactly as expected? If I understand it right, it should be: `(pos:complement(1123552..1123554),aa:Leu,seq:caa)`